src/services/user.js

```diff
@@ -7,10 +7,18 @@ import http from './http';
 import session from './session';
 import storage from './storage';
 
+/**
+ * @global
+ * @typedef User
+ * @property {int} current user's current cl&w credits
```

Comment:
JavaScript doesn't actually have an `int` type, so this and the line below it should be `Number` types.